configure.in.in

```diff
@@ -4,6 +4,9 @@
 @YAST2-INIT-COMMON@
 @YAST2-INIT-YCP@
 
+icondir=/usr/share/icons
+AC_SUBST(icondir)
+
```

Comment:
I'm just not sure if we need to do this in all YaST modules which use autotools.

We already have shared definitions in `yast2-devtools`, like we define `imagedir` ([here](https://github.com/yast/yast-devtools/blob/master/build-tools/aclocal/y2autoconf.m4#L45-L45) and [here](https://github.com/yast/yast-devtools/blob/master/build-tools/aclocal/y2autoconf.m4#L72)). I think it would make sense to add `icondir` there as well.

@mvidner ?

Reply:
:+1: 

I'd rather have one too many of those global definitions than one missing in weird scenarios.

Reply:
Well it also depends on whether this PR is a one-off or it's getting 100 little brothers soon.

Reply:
@mvidner sorry to inform, it already has 100 little brothers :/